### PR TITLE
CC-19: Added switch control on multiselect with tags component

### PR DESCRIPTION
**Pull request by vishesh-ats**
https://github.com/Components-care/backlog/issues/19

- Added knob(`Enable Switch`) on story to enable/disable switch on multi select with tags component

### CODE REVIEW

File: src/standalone/Selector/MultiSelectWithTags.tsx [hunk 1]
```diff
@@ -252,8 +351,6 @@ const MultiSelectWithTags = <Data extends MultiSelectData>(
 					<TextField
```
Comment (TQjona) [inline src/standalone/Selector/MultiSelectWithTags.tsx:351]: change this to a TextFieldWithHelp from UI-Kit so we can have consistent styles

File: src/standalone/Selector/MultiSelectWithTags.tsx [hunk 2]
```diff
@@ -90,8 +137,55 @@ const styles = createStyles((theme: Theme) => ({
...
+		float: "left",
+	},
+	switch: {
+		lineHeight: "30px",
+		float: "right",
+	},
 }));
 
+const AntSwitch = withStyles((theme: Theme) =>
+	createStyles({
```
Comment (TQjona) [inline src/standalone/Selector/MultiSelectWithTags.tsx:154]: inline styles don't need createStyles. createStyles is only needed if you want to store the JSS in a variable

File: src/standalone/Selector/MultiSelectWithTags.tsx [hunk 3]
```diff
@@ -74,6 +101,26 @@ export interface MultiSelectWithTagsProps<Data extends MultiSelectData>
 	 * Label for search input control
 	 */
 	searchInputLabel?: string;
+	/**
+	 * Display switch control
+	 */
+	displaySwitch?: boolean;
```
Comment (TQjona) [inline src/standalone/Selector/MultiSelectWithTags.tsx:107]: make switchLabel, switchValue and handleSwitch required if displaySwitch is true
```tsx
interface MultiSelectWithTagsSwitchPropsOn {
  displaySwitch: true;
  switchLabel: string;
  switchValue: boolean;
  handleSwitch: (...) => ;
}
interface MultiSelectWithTagsSwitchPropsOff {
  displaySwitch?: false;
}
type MultiSelectWithTagsSwitchProps = MultiSelectWithTagsSwitchPropsOn | MultiSelectWithTagsSwitchPropsOff;
interface Props extends MultiSelectWithTagsSwitchProps {
...
}
```

File: src/standalone/Selector/MultiSelectWithTags.tsx [hunk 4]
```diff
@@ -50,18 +65,30 @@ export interface MultiSelectWithTagsProps<Data extends MultiSelectData>
...
+	 */
+	defaultData: Data[];
 	/**
 	 * The currently selected values
 	 */
 	selected: Data[];
+	/**
+	 * Callback to set selected values
+	 */
+	setSelected?: (selectedValues: Data[]) => void;
```
Comment (TQjona) [inline src/standalone/Selector/MultiSelectWithTags.tsx:79]: seems unused

File: src/standalone/Selector/MultiSelectWithTags.tsx [hunk 5]
```diff
@@ -31,6 +38,14 @@ export interface MultiSelectData extends SelectorData {
 	canUnselect?: (data: MultiSelectData) => boolean | Promise<boolean>;
 }
 
+export interface Styles extends Partial<Record<SwitchClassKey, string>> {
+	focusVisible?: string;
+}
+
+export interface Props extends SwitchProps {
```
Comment (TQjona) [inline src/standalone/Selector/MultiSelectWithTags.tsx:45]: what's this? is it even being used somewhere?

**Review by TQjona — CHANGES_REQUESTED** [review]

**Comment by vishesh-ats** [comment]
@TQjona , Changes updated. Please Review.

**Review by TQjona — APPROVED** [review]

**Pull request merged**